components/combobox/data-bind.md

```diff
@@ -105,7 +104,7 @@ To bind the ComboBox to a model:
 
 ## Considerations
 
-The ComboBox component attempts to infer the type of its model and value based on the provided `Data` and initial `Value`. This affects the way its [reference is obtained](#component-reference) and what happens [if you can't provide data or a value](#missing-value-or-data). Providing a [value that is not in the data source](#value-out-of-range) needs to be taken into account be the app, because the component will not change it.
+The ComboBox component attempts to infer the type of its model and value based on the provided `Data` and initial `Value`. This affects the way its [reference is obtained](slug:common-features-data-binding-overview#component-type) and what happens [if you can't provide data or a value](#missing-value-or-data). Providing a [value that is not in the data source](#value-out-of-range) needs to be taken into account be the app, because the component will not change it.
```

Comment:
Double-check this fragment: `into account be the app`.

Reply:
There seem to be several instances of this phrase across the set. Address them all, if needed.

Reply:
It looks like a typo.